src/constants/markets.ts

```diff
@@ -219,3 +220,30 @@ export const ComingSoonMarkets: {[address: string]: ComingSoonMarketsData} = {
     isComingSoon: true,
   },
 }
+
+export type MarketDescriptionData = {
+  IndexesConstruction: Array<string>
+  FullLogo: string
+  ProposedBy: string
+  Approved: string
+  Contract: string
+  Sources: Array<string>
+  AdditionalInfo: string
+}
+
+export const MarketDescription: {[address: string]: MarketDescriptionData} = {
+  '111': {
+    IndexesConstruction: ['Nike Blahs', 'AirJordan 6', 'ASICS Retro', 'Puma Attack', 'Hoka Speedgoat'],
+    FullLogo: BtcdFull,
+    ProposedBy: '@unthoughtfulwizard',
```

Comment:
```suggestion
    ProposedBy: {name: '@unthoughtfulwizard', link: 'https://discord/twitter/etc'},
```
display name, and make it clickable to direct to link. @wloka-1 WDYT?